e2e/portal/tests/ProjectLevelAttachments/AddAttachmentsOnProjectLevel.spec.ts

```diff
@@ -0,0 +1,125 @@
+import { expect, Page, test } from '@playwright/test';
+import os from 'os';
+import path from 'path';
+
+import { SeedScript } from '@121-service/src/scripts/enum/seed-script.enum';
+import { resetDB } from '@121-service/test/helpers/utility.helper';
+
+import TableComponent from '@121-e2e/portal/components/TableComponent';
+import { generateLargeTestFile } from '@121-e2e/portal/helpers/largeFileGenerator';
+import LoginPage from '@121-e2e/portal/pages/LoginPage';
+import ProjectMonitoring from '@121-e2e/portal/pages/ProjectMonitoringPage';
+
+const pdfFilePath = path.resolve(
+  __dirname,
+  '../../../test-file-upload-data/test-document.pdf',
+);
```

Comment:
```suggestion
const pdfFilePath = path.resolve(
  '@121-e2e/test-file-upload-data/test-document.pdf',
);
```

Does this work? If not, it's fine as is, but also a bit ugly :/